packages/desktop-client/src/components/sidebar/Item.tsx

```diff
@@ -0,0 +1,89 @@
+import React, {
+  type CSSProperties,
+  type ComponentType,
+  type MouseEventHandler,
+  type ReactNode,
+  type SVGProps,
+} from 'react';
+
+// eslint-disable-next-line no-restricted-imports
+import { styles, colors } from '../../style';
+import Block from '../common/Block';
+import View from '../common/View';
+
+import { ItemContent } from './ItemContent';
+
+type ItemProps = {
+  title: string;
+  Icon: ComponentType<SVGProps<SVGElement>>;
+  Button?: ComponentType;
+  to?: string;
+  children?: ReactNode;
+  style?: CSSProperties;
+  indent?: number;
+  onClick?: MouseEventHandler<HTMLDivElement>;
+  forceHover?: boolean;
+  forceActive?: boolean;
+};
+export function Item({
+  children,
+  Icon,
+  Button,
+  title,
+  style,
+  to,
+  onClick,
+  indent = 0,
+  forceHover = false,
+  forceActive = false,
+}: ItemProps) {
+  const hoverStyle = {
+    backgroundColor: colors.n2,
+  };
+  const activeStyle = {
+    borderLeft: '4px solid ' + colors.p8,
+    paddingLeft: 19 + indent - 4,
+    color: colors.p8,
+  };
+  const linkStyle = {
+    ...styles.mediumText,
+    paddingTop: 9,
+    paddingBottom: 9,
+    paddingLeft: 19 + indent,
+    paddingRight: 10,
+    textDecoration: 'none',
+    color: colors.n9,
+    ...(forceHover ? hoverStyle : {}),
+    ':hover': hoverStyle,
+  };
+
+  const content = (
+    <View
+      style={{
+        flexDirection: 'row',
+        alignItems: 'center',
+        height: 20,
+      }}
+    >
+      <Icon width={15} height={15} />
+      <Block style={{ marginLeft: 8 }}>{title}</Block>
+      <View style={{ flex: 1 }} />
+      {Button && <Button />}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏looks like this `Button` prop is unused and can be removed instead of refactoring it from `button` -> `Button`.

Reply:
Removed :)